### DISCUSSION TITLE
posts/Go/bun/

### DISCUSSION THREAD

**Discussion by giscus** [Q1mi/BlogComments#372] [Announcements]
# posts/Go/bun/

李文周的 Blog 提供 Go 语言中文教程,李文周博客,Go 语言学习之路是我整理的一个免费教程，包含部分视频教程，也是非常适合新手学习Go语言的入门教程。本文介绍了SQL优先的 Go ORM 框架 bun 框架，介绍bun框架的优势及简单使用。

https://www.liwenzhou.com/posts/Go/bun/

**Comment by FlameCCG** [comment]
1

**Comment by FlameCCG** [comment]
12321312

**Comment by Verbsius** [comment]
顶一个
